handler/src/main/java/io/netty/handler/ssl/PemReader.java

```diff
@@ -96,7 +96,7 @@ static ByteBuf readPrivateKey(File file) throws KeyException {
 
         Matcher m = KEY_PATTERN.matcher(content);
         if (!m.find()) {
-            throw new KeyException("found no private key: " + file);
+            throw new KeyException("found no private PKCS8 key, maybe need to convert " + file + " to PKCS8 format ?");
```

Comment:
Didn't we decide to add a wiki page and add a link to the message?

    "could not find a PKCS #8 private key in a file: " + file + " (see http://netty.io/.... for more information)"